### ISSUE TITLE
spam in logs 

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#1401]
Hi, 
just notice this, I suppose anonymous shoud be able to do something with database. After I banned it, it change adress and start again. 2 days site live, latest github version. How to ban an range of adress? Asterix seems not work in ban user. 

![image](https://cloud.githubusercontent.com/assets/5429548/13426100/a45c98bc-dfab-11e5-8695-29c63c5623ff.png)
![image](https://cloud.githubusercontent.com/assets/5429548/13426174/172542ae-dfac-11e5-9218-b9978e45f422.png)

**Comment by lonalore** [comment]
Do not bann these IP addresses. These IPs belong normal (anonymous) users. 
The log-entries are generated by e107 after updating the number of "most ever online" (statistics).

**Comment by Jimmi08** [comment]
On other installation (older e107 version) there is nothing like this. And they are Chine adress.

**Comment by CaMer0n** [comment]
I believe this issue is due to storing IPs in IP6 format. Not sure how to get around it. Any ideas?

**Comment by Jimmi08** [comment]
Could you explain me what this mail means? I didn't find this in code and there is weird www.www. adress. It's spam IP adress, but I didn't find it in ban list. Thanks

![image](https://cloud.githubusercontent.com/assets/5429548/13547364/4bc4ddf4-e2d0-11e5-83d5-a1b1811e8c96.png)

**Comment by Moc** [comment]
@Jimmi08 I'm pretty sure this is a bounce email report. In other words: an email has been sent to one of your users (spam account), for example an email verification email, but it bounced back (e.g. emailadddress was incorrect/fake or mailserver was down, or the inbox was full, etc.).

**Comment by Moc** [comment]
Also, I think these are valid entries. Sure they are spambots but there is nothing wrong with the log itself. If you want, you can disable the logs in Tools > System logs > Preferences. I think it's the 'Login/Logout' option.

**Comment by Jimmi08** [comment]
@Moc thanks, but they are no other user then me. I see it's valid email but adress is not in database (maybe it's correct, I just don't know how this works in e107, in Nuke it was easy).
There is second problem, I wasn't able to find where (in what part of code) it was sended to check www adress. Because there is something wrong and if this is sended by e107, it could send wrong www with normal emails too. I have enabled Redirect all requests to site URL, but it did nothing, I needed to add it in htacess.   

I am really sorry to ask these questions but this is first site where I have problems with spams, banned IP because flood is really annoying and I am afraid if I didn't forget to set something, Now I will need to learn something about IP adress format, because they are saved differently than you see them in admin and your link produced different format too. Too long .htaccess can have influence on performance/speed and I have problems with speed (probably server/hosting related, it's changing during day).  

Thanks for your time
